microprofile/metrics/src/main/java/io/helidon/microprofile/metrics/RegistryProducer.java

```diff
@@ -40,7 +41,8 @@ private RegistryProducer() {
     @Produces
     @Default
     public static org.eclipse.microprofile.metrics.MetricRegistry getScopedRegistry(InjectionPoint injectionPoint) {
-        RegistryScope scope = injectionPoint.getAnnotated().getAnnotation(RegistryScope.class);
+        Annotated annotated = injectionPoint == null ? null : injectionPoint.getAnnotated();
+        RegistryScope scope = annotated == null ? null : annotated.getAnnotation(RegistryScope.class);
```

Comment:
Minor, parentheses around condition help readability in this case IMO.

Reply:
Changed